notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: "takes the exponential of the tangent vectorat the point p"</p><p> </p><p>NIT: 1) identifying the unique geodesic gamma</p><p>--&gt; gamma(t) represents a point on the geodesic (the point corresponding to parameter t), gamma represents the whole curve</p><p> </p><p>Definition of the exponential: I would refrain from using the notion of "length", because we do not have this concept yet (Riemannian metric has not been introduced).</p><p>Thus:</p><p>2) calculate another point p' that corresponds to gamma(1), i.e. to the point reached at time t=1 on the geodesic gamma.</p><p> </p><p>NIT: missing period at the end of the sentence</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>hmmm i think that using "times" might be confusing. Do you think that it would be okay to keep the "distance" part but add in a note about how we do not yet know what distance means because we haven't discussed metrics?</p>

Reply:
<p>For example: Note again that the notion of "length" is different on a manifold than it is in Euclidean space and that quantifying length is not something that we will be able to do without specifying a metric (discussed in <code>RiemannianMetric</code> section)&nbsp;</p>